CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Changelog
 All changes to the project will be documented in this file.
 
+## [1.37.17]
```

Comment:
In some issue we agreed to not write "no released yet" or the date, because we don't know when it will actually release, and the not released yet shows up in VSCode extension readme if we release with it in place.

Reply:
this applies to the VS Code changelog, not Omnisharp server release notes - those are independent

Reply:
Right, my bad.